diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 4d9f48a17292b2150d77f4b75946b5631af68a33..f6a8d5f0c63941231e7371ba582d257e25bb8cdc 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1273,6 +1273,7 @@ static_idl_files_in_modules = [
   "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
   "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
 ]
+static_idl_files_in_modules += brave_static_idl_files_in_modules
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
