diff --git a/BUILD.gn b/BUILD.gn
index 00baa79737b9c159e6aba990eed898a3f66a250b..8b4b6b7fbc349b4865b7baedc0057bf12cdc52fe 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -784,6 +784,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3267,6 +3268,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
